app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,71 @@
+class RentalsController < ApplicationController
+  def index
+    rentals = Rental.all
+    render json: rentals.as_json(), status: :ok
+  end
+
+def check_out
+  customer = Customer.find_by(id: params[:customer_id])
+  video = Video.find_by(id: params[:videos_id])
+  rental = Rental.new(customer_id: params[:customer_id], videos_id: params[:videos_id], due_date: (Date.today + 7))
+  if video.nil? || customer.nil? || video.available_inventory < 1
+    render json: {
+      errors: ['Not Found']
+    }, status: :not_found
+    return
+  end
+
+  if rental.save
```

Comment:
What about if `rental.save` returns `false`?  You should have some kind of response with an error response code here.